services/src/main/java/io/scalecube/services/ServiceHeaders.java

```diff
@@ -30,40 +30,36 @@
    * header is used to mark a request as a service response.
    */
   public static final String SERVICE_RESPONSE = "service-response";
-
-  private ServiceHeaders() {
-    // Do not instantiate
```

Comment:
Why did you remove this constructor it seems you don't want to instantiate this class?